play28-core/src/main/scala/play/api/test/ops/AsyncResultExtractors.scala

```diff
@@ -0,0 +1,104 @@
+package play.api.test.ops
+
+import akka.stream.Materializer
+// Despite the package name this is from play-test
+import akka.stream.testkit.NoMaterializer
```

Comment:
This is the only change from the 2.7 version I reused the implementation of this from `play-test` because the signature had changed because it had to extend private methods.